js/sia.js

```diff
@@ -189,7 +207,7 @@ function SiadWrapper () {
    * @returns {object} siad configuration object
    */
   function configure (settings, callback) {
-    for (var key in settings) {
+    for (let key in settings) {
```

Comment:
`let` has smaller scope than `var`. `let`'s scope is the enclosing block (`for` in this case) and `var`'s scope is the enclosing function. So `key` won't be available outside the loop, sort of like `for i := range foo` in Go.